### PR TITLE
Agummybear/add reset command

### PR DESCRIPTION
**Pull request by AGummyBear**
This PR adds a ROS service to the vectornav IMU node that allows you to send a `reset()` command to the device.

If the device is reset, the node will re-connect the same way that it did initially and resume publishing IMU data.

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -149,115 +159,126 @@ int main(int argc, char *argv[])
...
-        vs.setResponseTimeoutMs(1000); // Wait for up to 1000 ms for response
-        vs.setRetransmitDelayMs(50);  // Retransmit every 50 ms
-
-        // Acceptable baud rates 9600, 19200, 38400, 57600, 128000, 115200, 230400, 460800, 921600
-        // Data sheet says 128000 is a valid baud rate. It doesn't work with the VN100 so it is excluded.
-        // All other values seem to work fine.
-        try{
-            // Connect to sensor at it's default rate
-            if(defaultBaudrate != 128000 && SensorBaudrate != 128000)
+    while (ros::ok())
```
Comment (Spwizzard) [inline src/main.cpp:162]: did you actually change anything in here or is github just being dumb?

Comment (Spwizzard) [reply src/main.cpp:162]: besides the while(ros::ok()) loop

Comment (AGummyBear) [reply src/main.cpp:162]: I ended up wrapping a huge block of code (all the connection and initialization steps) inside a new while() loop, which allows the node to re-connect properly in the event that the sensor is reset. But yeah, the diff looks really ugly.

Comment (Spwizzard) [reply src/main.cpp:162]: That's what I figured but I was struggling to determine if there were other changes. Sounds good.

**Pull request merged**